cpp/src/arrow/flight/flight_test.cc

```diff
@@ -374,6 +374,24 @@ TEST(TestFlight, ServeShutdown) {
   }
 }
 
+TEST(TestFlight, ServeShutdownWithDeadline) {
+  constexpr int kIterations = 10;
+  for (int i = 0; i < kIterations; i++) {
```

Comment:
Is it actually useful to do this in a loop?

Reply:
I'm modeling this on the test above (`ServeShutdown`). Not sure if it's useful.

Reply:
I don't think the so. The test above is a bit more involved.

Reply:
Removed the loop.